doc/filters/striptags.rst

```diff
@@ -13,3 +13,11 @@ by one space:
     Internally, Twig uses the PHP `strip_tags`_ function.
 
 .. _`strip_tags`: http://php.net/strip_tags
+
+    As a consequence, a second parameter, `$allowable_tags`, can be provided. In this example, the tags like `<br/>`, `<br>`, `<p>` and `</p>` will remain in the string:
```

Comment:
this should be above the list of links (which should stay at the bottom of the file)